python/ray/autoscaler/v2/instance_manager/instance_manager.py

```diff
@@ -67,251 +63,3 @@ def get_instance_manager_state(
             The reply to the request.
         """
         pass
-
```

Comment:
move to prevent circular dependency with `InstanceUtil` and `AutoscalingInstance`